data/load-seed-data.js

```diff
@@ -11,26 +11,26 @@ async function run() {
   try {
     await client.connect();
 
-    const users = await Promise.all(
-      usersData.map(user => {
-        return client.query(`
-                      INSERT INTO users (email, hash)
-                      VALUES ($1, $2)
-                      RETURNING *;
-                  `,
-        [user.email, user.hash]);
-      })
-    );
+    // const users = await Promise.all(
+    //   usersData.map(user => {
+    //     return client.query(`
+    //                   INSERT INTO users (email, hash)
+    //                   VALUES ($1, $2)
+    //                   RETURNING *;
+    //               `,
+    //     [user.email, user.hash]);
+    //   })
+    // );
       
-    const user = users[0].rows[0];
+    // const user = users[0].rows[0];
 
     await Promise.all(
-      animals.map(animal => {
+      powers.map(powers => {
         return client.query(`
-                    INSERT INTO animals (name, cool_factor, owner_id)
-                    VALUES ($1, $2, $3);
+                    INSERT INTO powers (id, name, description, realistic, type)
```

Comment:
Ahh, close here! You don't actually want to INSERT the `id` in your SQL query since this happens automatically by SQL. You'll just want to add the properties that you are tracking yourself (i.e name, description....etc.).